eng/Versions.props

```diff
@@ -46,6 +46,6 @@
     <!-- Standard dependencies -->
     <NETStandardLibraryPackageVersion>2.1.0-prerelease.19158.1</NETStandardLibraryPackageVersion>
     <!-- dotnet-optimization dependencies -->
-    <optimizationwindowsntx64IBCCoreFxPackageVersion>99.99.99-master-20190227.1</optimizationwindowsntx64IBCCoreFxPackageVersion>
+    <optimizationwindows_ntx64IBCCoreFxPackageVersion>99.99.99-master-20190308.5</optimizationwindowsntx64IBCCoreFxPackageVersion>
```

Comment:
Was renaming the property intentional? If yes you also need to change the closing element.

Reply:
Yes. Maestro hasn't been updating the package version because this was incorrect. Will fix. Thanks